module/os/linux/zfs/vdev_disk.c

```diff
@@ -197,12 +211,47 @@ vdev_disk_kobj_evt_post(vdev_t *v)
 	}
 }
 
+#if !defined(HAVE_BLKDEV_GET_BY_PATH_4ARG)
+/*
+ * Define a dummy struct blk_holder_ops for kernel versions
+ * prior to 6.5.
+ */
+struct blk_holder_ops {};
+#endif
+
+static inline struct block_device *
```

Comment:
```suggestion
static struct block_device *
```

These can just be `static`, originally I suggested `static inline` since I thought you might be added them to the compat header.